pkg/agent/ngt/handler/grpc/handler.go

```diff
@@ -87,9 +83,8 @@ func (s *server) SearchByID(ctx context.Context, req *payload.Search_IDRequest)
 func toSearchResponse(dists []model.Distance, err error) (res *payload.Search_Response, rerr error) {
 	res = new(payload.Search_Response)
 	if err != nil {
-		// TODO: not found error or others
 		log.Errorf("[toSearchResponse]\tUnknown error\t%+v", err)
-		return nil, status.WrapWithUnknown("Unknown error occurred", &errDetail{method: "Search"}, err)
+		err = status.WrapWithInternal("Search API error occurred", err, info.Get())
```

Comment:
![badge](https://img.shields.io/badge/review-note-informational.svg?style=flat-square)
Thanks. It's better than the previous one.